### PR TITLE
feat: add raycast

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      add doc     |
| 🇨🇳 Chinese |     添加文档      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by crazyair** [comment]
image

File: components/overview/index.zh-CN.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ title: 组件总览
 type: 组件总览
 ---
 
-`antd` 为 Web 应用提供了丰富的基础 UI 组件，我们还将持续探索企业级应用的最佳 UI 实践。除了官方组件，我们也提供了[社区精选组件](/docs/react/recommendation)作为必要的补充，另外如果您是内网用户，欢迎尝试使用 [TechUI](https://techui.alipay.com)。
+`antd` 为 Web 应用提供了丰富的基础 UI 组件，我们还将持续探索企业级应用的最佳 UI 实践。除了官方组件，我们也提供了[社区精选组件](/docs/react/recommendation)作为必要的补充，另外如果您是内网用户，欢迎尝试使用 [TechUI](https://techui.alipay.com)。`mac`用户可安装[Raycast](https://www.raycast.com/)软件，并安装插件[antd-open-browser](https://www.raycast.com/crazyair/antd-open-browser)实现快速打开组件。
```
Comment (zombieJ) [inline components/overview/index.zh-CN.md:7]: 加到这个页面~
https://ant.design/docs/resources-cn

File: docs/resources.zh-CN.md [hunk 2]
```diff
@@ -83,6 +83,10 @@ toc: false
   - https://mastergo-local-default.oss-cn-beijing.aliyuncs.com/ant-design-mastergo.svg
   - 可在「MasterGo」在线免费使用的全套组件和模板
   - https://mastergo.com/community/?utm_source=antdesign&utm_medium=link&utm_campaign=resource&cata_name=AntDesign
+- 快速打开 Ant Design 组件
+  - https://gw.alipayobjects.com/zos/basement_prod/5edc7f4d-3302-4710-963b-7b6c77ea8d06.svg
+  - mac 用户可使用 Raycast 快速打开 Ant Design 组件
+  - https://www.raycast.com/crazyair/antd-open-browser
```
Comment (afc163) [inline docs/resources.zh-CN.md:89]: 这里是放设计资源和工具的，开发工具不要放在这里。

Comment (crazyair) [reply docs/resources.zh-CN.md:89]: 一开始是放在这里
image

**Pull request merged**